src/lib/allDbTypes.d.ts

```diff
@@ -0,0 +1,32 @@
+// This is the one place allowed to import from generated types as here is were we re-export them
```

Comment:
suggestion: could the `lib` folder be renamed `database`?

Reply:
Makes sense will change.
Subsequent PR might wanna mess around with folder structure more, so might reach out abt that